MLS.Agent/CommandLine/KernelServerCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CommandLine;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive;
+using WorkspaceServer.Kernel;
+
+namespace MLS.Agent.CommandLine
+{
+    public static class KernelServerCommand
+    {
+        public static async Task<int> Do(
+            IConsole console)
+        {
+            var kernel = new CompositeKernel()
+            {
+                  new CSharpKernel()
```

Comment:
We should share the code that sets up the kernel extensions, etc. with the Jupyter mode startup.